website/language.py

```diff
@@ -14,7 +14,7 @@
 # (upon clicking primary email confirmation link)
 WELCOME_MESSAGE = '''
 <h1>Welcome to the OSF!</h1>
-<p>Visit our <a href="[https://osf.io/getting-started/">getting started page</a> to learn about creating a project, or get inspiration from <a href="https://osf.io/explore/activity/#popularPublicProjects">popular public projects</a></p>
+<p>Visit our <a href="https://osf.io/getting-started/">getting started page</a> to learn about creating a project, or get inspiration from <a href="https://osf.io/explore/activity/#popularPublicProjects">popular public projects</a></p>
```

Comment:
I'd suggest changing "get inspiration" to "find inspiration", but I assume this text was decided in the proposal. @erinmayhood ?